src/pages/airport/[icao].vue

```diff
@@ -351,7 +352,12 @@ const aircraftModes: SelectItem<MapAircraftMode>[] = [
     },
 ];
 
-const displayedColumns = ref<MapAircraftKeys[]>([]);
+const displayedColumns = useCookie<MapAircraftKeys[]>('dashboard-displayed-columns', {
```

Comment:
Please remove useCookie and go back to array mode. We'll have presets for this in 0.5.0 hopefully

Reply:
Do I keep the selected by default feature, but with array mode?

Reply:
Sure! Just update array with same keys as you did here in default